src/utils/CategoryUtils/CategoryUtils.ts

```diff
@@ -0,0 +1,113 @@
+import {CustomIconNames} from '../../components';
+import {Constants} from '../../constants';
+import {CategoryInterface} from '../../models';
+import {FeaturedCategories, MoreCategories} from './CategoryUtils.types';
+
+const featuredCategories = (categories: CategoryInterface[]) => {
+  const topFeaturedItems = [
+    'restaurants',
+    'shopping',
+    'nightlife',
+    'active',
+    'beautysvc',
+    'homeservices',
+  ];
+
+  const moreItemArray = [
+    {
+      alias: 'more',
+      title: 'More',
+      parent_aliases: [],
+      country_blacklist: [],
+      country_whitelist: [],
+    },
+  ];
+
+  const featuredCategoryItems = categories.filter(category =>
+    topFeaturedItems.includes(category.alias),
+  );
+
+  const extraCategoryItems = categories
+    .filter(category => category.parent_aliases.length === 0)
+    .filter(category => !featuredCategoryItems.includes(category))
+    .slice(0, 7 - featuredCategoryItems.length);
```

Comment:
Create const for the magic number 7 explaining what it is

Reply:
Added Constants